### DISCUSSION TITLE
Is this an expected behavior?

### DISCUSSION THREAD

**Discussion by nputhuva-rm** [SafeExamBrowser/seb-win-refactoring#404] [Q&A] [answered]
Configured Quit/Unlock password in the SEB file
Configure close of the lid to sleep mode for the device.
Launch the SEB Browser and Login as student to the exam and tried the below scenarios.
Action 1: Close & open the lid of the device. Login to SEB launched user account
Result: RED screen is displayed only once prompting for exit code
Action 2: Close & open the lid of the device. Login to Guest user account and return to SEB launched user account by logging off from Guest Account.
Result: RED screen is displayed only once prompting for exit code
Action 3: Close & open the lid of the device. Login to Guest account and return to SEB launched account with out logging off from Guest account
Result: RED screen is displayed twice prompting for exit code as below:
MS Edge browser :
 
![image](https://user-images.githubusercontent.com/104403591/165419833-05ebdd52-f6e2-461f-923d-542058036d65.png)

normal lock screen: 
 
![image](https://user-images.githubusercontent.com/104403591/165419883-761da076-bf2b-4477-9ee1-579ccaf3f960.png)

Action4: Close & open the lid of the device. Login to Guest account and open the browser(Chrome, Firefox or Edge), return to SEB launched account with out logging off from Guest account
Result: RED screen is displayed twice prompting for exit code as below:
MS Edge browser
normal lock screen
Action 5: Close & open the lid of the device. Login to Guest account and open the browser(Chrome, Firefox or Edge), return to SEB launched account by logging off from Guest account
Result: RED screen is displayed only once prompting for exit code
Action 6: Close & open the lid of the device. Login to Guest account and open the browser(Chrome, Firefox or Edge), return to SEB launched account with out logging off from Guest account. Select exit browser in the RED screen. Try to login back to the exam
Result: RED screen is displayed twice prompting for exit code as below:
MS Edge browser
normal lock screen
If the student tried to launch the SEB config file, system displays the prompt to close the Edge browser before launching the Brower. system displays the error message as 
 
![image](https://user-images.githubusercontent.com/104403591/165419927-27da23be-5a75-411f-9baa-5ff8c5f4faf8.png)

Query: **_Can i get a confirmation whether these are the Expected behavior?_**  @dbuechel

**Comment by dbuechel** [comment, marked as answer]
Yes, at the moment there are two lock screen scenarios implemented, and if you trigger both of them (starting of a prohibited application which can't automatically be terminated and user switch), then currently both of them get rendered. Also the error message during startup is as intended, as a student can't be allowed to start a new session if a prohibited application can't be terminated automatically.
